src/manager.cpp

```diff
@@ -164,55 +167,102 @@ sdbusplus::async::task<> Manager::startSyncEvents()
     co_return;
 }
 
-sdbusplus::async::task<bool>
-    // NOLINTNEXTLINE
-    Manager::syncData(const config::DataSyncConfig& dataSyncCfg,
-                      fs::path srcPath)
+void Manager::getRsyncCmd(RsyncMode mode,
+                          const config::DataSyncConfig& dataSyncCfg,
+                          const std::string& srcPath, std::string& cmd)
 {
     using namespace std::string_literals;
 
-    // For more details about CLI options, refer rsync man page.
-    // https://download.samba.org/pub/rsync/rsync.1#OPTION_SUMMARY
-    std::string syncCmd{
-        "rsync --compress --recursive --perms --group --owner --times --atimes"
-        " --update --relative --delete --delete-missing-args "};
+    if (mode == RsyncMode::Sync)
+    {
+        // Appending required flags to sync data between BMCs
+        // For more details about CLI options, refer rsync man page.
+        // https://download.samba.org/pub/rsync/rsync.1#OPTION_SUMMARY
 
-    if (dataSyncCfg._excludeList.has_value())
+        cmd.append(
+            "rsync --compress --recursive --perms --group --owner --times --atimes"
```

Comment:
All these common option could be in default and append necessary option `--remove-source-files` if the mode is notify to avoid updating command in two different places.